### PR TITLE
Macbook finds sensor with these changes

### PR DESCRIPTION
**Pull request by antonmc**
This gets me going with SensorTag 2 and a macbook ... can't say what it means for previous version of the sensor tag.

### CODE REVIEW

File: publish/README.md [hunk 1]
```diff
@@ -32,8 +32,15 @@ the steps below to register your device.
 
 After clicking Continue you will be brought to a page containing some properties for the device you registered.
 Copy the properties into a file called config.properties in the publish directory of this repo.
-Make sure you do this before you leave this page, you will not be able to retrieve these properties after you leave the 
-page.
+Make sure you do this before you leave this page, you will not be able to retrieve these properties after you leave the page.
+
+config.properties example - provide your own ids, tokens etc:
+
```
Comment (ryanjbaxter) [inline publish/README.md]: @antonmc can you surround this in the markdown syntax for code?

**Comment by ryanjbaxter** [comment]
@antonmc thanks so much for this!  I confirmed the updated dependencies work with both the new and old sensor tag!  Just see my one comment and I will merge in the changes.

**Comment by ryanjbaxter** [comment]
I also tested this on the Beaglebone Black and it worked fine there as well

**Comment by antonmc** [comment]
Great. I've added code comments in the readme. On a trip home ( typing from a departure lounge ). I'm working on a couple of new interaction visualizations if you're interested. Share them soon.

**Comment by ryanjbaxter** [comment]
Thanks @antonmc!

**Pull request merged**